comp/core/configsync/configsyncimpl/params.go

```diff
@@ -0,0 +1,26 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2024-present Datadog, Inc.
+
+// Package configsyncimpl implements synchronizing the configuration using the core agent config API
+package configsyncimpl
+
+import "time"
+
+// Params defines the parameters for the configsync component.
+type Params struct {
+	SyncTO     time.Duration
+	SyncDelay  time.Duration
+	SyncOnInit bool
```

Comment:
`SyncTO` make it sound it's a destination (ie sync to something).

Since we're in the `configsync` compo we should get rid of the `Sync` prefix.
```suggestion
	Timeout    time.Duration
	Delay    time.Duration
	OnInit bool
```

Reply:
Yeah, that makes sense.